### ISSUE TITLE
dial tcp :0: connection refused

### ISSUE THREAD

**Issue by tkisme** [drone/drone#1251]
This error is not like
```
dial tcp 0.0.0.0:4243: connection refused
```
It is strange 
```
dial tcp :0: connection refused
```

**Comment by bradrydzewski** [comment]
Can you provide some more details regarding what version of Drone you are running? How are you running it (in a container, debian file, etc). Can you provide an example of your configuration?

**Comment by tkisme** [comment]
already fixed in 0.4
